.github/workflows/run-integration.yml

```
@@ -0,0 +1,155 @@
+#     empty and accumulates. Reading it as truncating would make the bundle
+#     imports look ungated, which is the frightening and wrong conclusion.
+#   * the suites passed -> the pytest call in
+#     scripts/automated_tests/opus_run_unittests_coverage.sh
+#   * coverage is 100% -> scripts/automated_tests/opus_check_coverage.sh, the
+#     "Check coverage level" step below. opus_main_test.sh does NOT call it: the
+#     check is deliberately after the codecov upload so a coverage failure still
+#     reports.
+name: Run Integration Tests
+run-name: "Run Integration Tests: ${{ github.ref_type }} ${{ github.ref_name }} by ${{ github.triggering_actor }}"
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)